packages/cli/src/WebhookServer.ts

```diff
@@ -217,6 +218,8 @@ class App {
 
 		this.presetCredentialsLoaded = false;
 		this.endpointPresetCredentials = config.getEnv('credentials.overwrite.endpoint');
+
+		initErrorHandling(this.app);
```

Comment:
Some errors are not captured in their try-catch blocks because they are captured elsewhere, as in this case. This reduces the number of calls but makes it possible to forget to capture errors where this does not apply. Is this programmatically detectable so we can lint for it? (And exempt when we know this is reported elsewhere.)

Reply:
we don't need to worry about this as all unhandled errors are eventually caught either via `uncaughtException` or the `unhandledRejection` event on the process, and those are already being handled by Sentry.